onnxruntime/test/providers/cpu/nn/conv_transpose_op_test.cc

```diff
@@ -533,5 +532,25 @@ TEST(ConvTransposeTest, ConvTranspose_DefaultStridesAndDilations) {
   TestConvTransposeOp(attrs, {X, W}, {X_shape, W_shape}, expected_vals, Y_shape);
 }
 
+TEST(ConvTransposeTest, ConvTranspose_1D_NonDefaultStridesAndDilations) {
+  ConvTransposeOpAttributes attrs = {
+      vector<int64_t>{4},     // kernel_shape
```

Comment:
@lara-hdr - I couldn't use your test directly as the input shape in that test was 1, 1, 50, 50 and it is going to be hard to store 2500 values in the same unit file. I crafted a smaller sized input exhibiting the same phenomenon.